codebase/superdarn/src.lib/tk/fitacf_v3.0/src/determinations.c

```diff
@@ -527,7 +531,40 @@ void set_nump(llist_node range, struct FitRange* fit_range_array){
  * @param      fit_prms         The FITPRM struct holding rawacf record info.
  *
  */
-void find_elevation(llist_node range, struct FitData* fit_data, FITPRMS* fit_prms){
+void find_elevation(llist_node range, struct FitData* fit_data, FITPRMS* fit_prms, int elv_version){
+
+    double azi_offset,phi_0;
+    RANGENODE* range_node;
+    range_node = (RANGENODE*) range;
+   
+    azi_offset = fit_prms->maxbeam/2 - 0.5;
+    phi_0 = fit_prms->bmsep * (fit_prms->bmnum - azi_offset) * PI/180;
+
+    // elv_version 2 is Simon's
+    // elv_version 1 is original
+    // elv_version 0 is goosebay
```

Comment:
Just some clarification for future developers: 

```suggestion
    // elv_version 2 is the Shepherd [2017] elevation calculation
    // elv_version 1 is original elevation calculation
    // elv_version 0 is specifically for the GBR radar when -old_elev is specified
```